shared/src/test/scala/pl/metastack/metatime/SchedulerTest.scala

```diff
@@ -41,8 +40,7 @@ class SchedulerTest extends AsyncFunSuite {
     val timeBefore = Time.now()
     val p = Promise[Boolean]()
     scheduler.every(Time(0, 0, 1, 0)) {
-      // TODO Check whether every() is called more than once in the same intervals
```

Comment:
Please keep this comment because we don't check yet whether `every()` is correctly implemented.